http/cookie_map.ts

```diff
@@ -607,9 +607,11 @@ export class SecureCookieMap extends CookieMapBase {
   /** Sets all cookies in the {@linkcode Request} to be deleted in the
    * response. */
   async clear(options: SecureCookieMapSetDeleteOptions) {
+    const promisesCookiesKeys: Promise<this>[] = [];
```

Comment:
```suggestion
    const promises = [];
```
Ditto. This, I think, would've sufficed.